pkg/cmd/host_test.go

```diff
@@ -71,10 +87,10 @@ var hostRotateAPIKeyCmdTestCases = []struct {
 	},
 }
 
-func TestHostRotateAPIKeyCmd(t *testing.T) {
-	for _, tc := range hostRotateAPIKeyCmdTestCases {
+func TestrotateHostAPIKeyCmd(t *testing.T) {
```

Comment:
I was wondering why CodeClimate showed a line coverage decrease and when I opened this file locally it warned that the character after "Test" can't be lowercase
```suggestion
func TestRotateHostAPIKeyCmd(t *testing.T) {
```

Reply:
Thanks. Nice catch. I was also confused about the coverage